app/views/handlers/confirmation_email.py

```diff
@@ -63,16 +65,29 @@ def get_page_title(self):
         return self.page_title
 
     def _publish_fulfilment_request(self):
-        topic_id = current_app.config["EQ_SUBMISSION_CONFIRMATION_TOPIC_ID"]
-        fulfilment_request = ConfirmationEmailFulfilmentRequest(
+        fulfilment_request = ConfirmationEmailTask(
             self.form.email.data, self._session_store.session_data, self._schema
         )
+
+        task = {
+            "http_request": {
+                "http_method": HttpMethod.POST,
+                "url": "",
+                "oidc_token": {"service_account_email": ""},
+                "headers": {
+                    "Content-type": "application/json",
+                },
+                "body": fulfilment_request.message,
+            },
+        }
```

Comment:
How come this was extracted out?. The struct is very specific to task creation. I think the url and email being done in create_task and the rest here isn't very clear.
I think we should extract the whole things out or none of it.|

Thoughts?

Reply:
As discussed I've moved the additional bits out, using `current_app.eq["cloud_tasks"].project_id` to get the project id for now but can change this pending further discussions

Reply:
As discussed, was moved back to the cloud task publisher file